remotesigning/validation.go

```diff
@@ -89,3 +97,161 @@ func CalculateWitnessHashPSBT(transaction string) (*string, error) {
 	result := hex.EncodeToString(hash)
 	return &result, nil
 }
+
+func validateScript(signing *SigningJob, seedBytes []byte) bool {
+	hardenedPath, _, err := SplitDerivationPath(signing.DerivationPath)
+	if err != nil {
+		log.Printf("Failed to parse derivation path (Previous Transaction): %v", err)
+		return false
+	}
+	_, remainingPathChangeAddress, err := SplitDerivationPath(signing.L1DerivationPath)
+	if err != nil {
+		log.Printf("Failed to parse derivation path (Next Transaction): %v", err)
+		return false
+	}
+
+	// Step 1: Derive Tx Script from Master Seed
+	masterSeedHex := hex.EncodeToString(seedBytes)
+	xpub, err := utils.GenHardenedXPub(masterSeedHex, hardenedPath, "mainnet")
+	if err != nil {
+		log.Printf("Failed to generate base xpub: %v", err)
+		return false
+	}
+	childPubkey, err := utils.DeriveChildPubKeyFromExistingXPub(xpub, remainingPathChangeAddress)
+	if err != nil {
+		log.Printf("Failed to derive pubkey: %v", err)
+		return false
+	}
+	generated_script, err := GenerateScriptFromChildPubkey(childPubkey)
+	if err != nil {
+		log.Printf("Failed to generate script from seed: %v", err)
+		return false
+	}
+	log.Printf("✓ Successfully generated script from seed! %x", generated_script)
```

Comment:
Will defer to @zhenlu as to whether we need all these logs if we already have the error logs, not sure what's typical in our SDKs